.vscode/settings.json

```diff
@@ -1,18 +1,7 @@
 {
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true
+  },
   "eslint.enable": true,
-  "eslint.autoFixOnSave": true,
-  "eslint.validate": [{
```

Comment:
don’t we need to do this?

“eslint.validate": ["javascript", ..., “typescript”]

Reply:
`eslint.validate` is deprecated and `eslint.probe` should be used instead. `eslint.probe` defaults to supporting js and ts so we don't need to manually configure it.